test/unit/Controllers/UserController.test.ts

```diff
@@ -35,4 +35,38 @@ describe("UserController", function () {
             }
         });
     });
+
+    describe('POST /user/login', () => {
+        const userServiceStub = sinon.createStubInstance(UserService);
+        // @ts-ignore
+        userServiceStub.getUserByEmail.resolves({
+            email: 'example@example.com',
+            // eslint-disable-next-line
+            hash_password: '$2a$10$/ZUT2CX0acd/ZgCHT8apxOB1yxYhzTkx3bzi5YKzKFn78vm2RQhxi'
+        });
+
+        it('should return JWT on existing valid email and password input', async function () {
```

Comment:
When you move that part above to service you can test the service with all cases (both positive and negative).